src/components/VoteBox.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import styled from "styled-components";
+export default function VoteBox() {
+  const [voteDataList, setVoteDataList] = useState([]);
+
+  const sortVoteDataList = (dataList) => {
+    const returnDataList = dataList
+      .concat()
+      .sort((a, b) => (a.voteCount > b.voteCount ? -1 : 1));
+    return returnDataList;
+  };
+
+  useEffect(() => {
+    const fetchVoteData = async () => {
+      try {
+        const response = await axios.get(
+          "http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates"
+        );
+        const initialVoteDataList = sortVoteDataList(response.data);
+        setVoteDataList(initialVoteDataList);
+      } catch (e) {
+        console.log(e);
+      }
+    };
+    fetchVoteData();
+  }, []);
+
+  const sendVoteResult = async (id, name) => {
+    try {
+      const response = await axios.get(
+        `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${id}`
+      );
+      //아 get으로도 보내는건가..? post를 안써도 되는거야?
+      if (response) {
+        alert(`${name}님께 투표 완료!`);
+      }
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const handleVoteButton = (id, name) => {
+    sendVoteResult(id, name);
+    const nextVoteDataList = voteDataList.map((data, index) => {
+      if (data.id !== id) return data;
```

Comment:
`try, catch, if` 섞어서 쓰기보다 이렇게 써도 좋을 것 같아용🤗

```suggestion
  const handleVoteCount = async (id, name) => {
    await axios
    .get(
        `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${id}`)
    .then((response) => {
	console.log(response);
        alert(`${name}님께 투표 완료!`);
      })
    .catch((error) => {
	console.log(error);
	alert('투표 실패!');
	});
    }
```

Reply:
.then .catch를 이용하니 확실히 가독성이 더 좋아지네요!! 👍